packages/automerge-repo/src/storage/StorageSubsystem.ts

```diff
@@ -122,18 +122,18 @@ export class StorageSubsystem {
     if (!this.#shouldSave(documentId, doc)) {
       return
     }
-    let sourceChunks = this.#chunkInfos.get(documentId) ?? []
+    const sourceChunks = this.#chunkInfos.get(documentId) ?? []
     if (this.#shouldCompact(sourceChunks)) {
-      this.#saveTotal(documentId, doc, sourceChunks)
+      void this.#saveTotal(documentId, doc, sourceChunks)
```

Comment:
what's with the voids?

Reply:
These are all hanging promises, and my understanding is that this is the best way to indicate that you are not doing anything with the result

Reply:
I think if we're going to do this we should enable a linter rule that enforces it or else we should not do it at all. I have a mild preference for the latter (the voids are just another thing to read and understand IMHO) but this is a very weak opinion and I'd be fine with having them everywhere. I'd like to avoid it becoming a thing we have to *remember* to do and so only some files have them and others not.

Reply:
I'm going to merge this since the rest of it is objectively better and this is fine but I would appreciate it if you'd pick either one of the two options and run with it :)

Reply:
Interestingly, the reason I spotted these was because my editor already flags these. Will look into one the why and make it explicit either way

Reply:
Seems like it's already in the eslint rules. Does it not show an error on your end?

https://github.com/automerge/automerge-repo/blob/3995f1fbe8f5fed3aa4236bfd8bad61c416675dc/packages/automerge-repo/.eslintrc#L23C37-L23C37